controller/src/main/resources/org/jboss/as/controller/descriptions/common/LocalDescriptions.properties

```diff
@@ -430,6 +430,7 @@ global.read-attribute-group.resolve-expressions=Resolves expressions to current
 global.read-attribute-group-names=Gets the names of all the attribute groups under the selected resource
 global.read-children-names=Gets the names of all children under the selected resource with the given type
 global.read-children-names.child-type=The name of the node under which to get the children names
+global.read-children-names.include-singletons=If 'true' include the allowed values for any singleton registration.
```

Comment:
If 'true' include the allowed values for any singleton registrations, even if no resource currently exists with thet name.